db/model/botAction.js

```diff
@@ -18,8 +18,7 @@ const assoc = {};
 const dbErrors = require('../dbErrors');
 const constants = require('../constants');
 const u = require('../helpers/botUtils');
-const looksLikeId =
-  /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
+const v = require('../../utils/common');
```

Comment:
"v" as the constant name made sense when it was "verbs" but not so much any more